docs/reference/contracts/apis/managed.md

```diff
@@ -0,0 +1,294 @@
+# Managed Pools
+
+## BPT Supply
+
+### `getActualSupply`
+
+```solidity
+getActualSupply() 
+returns (uint256)
+```
+
+Returns the effective BPT Supply. Implemented in `ManagedPoolSettings`. 
+
+::: warning
+Since this function reads balances directly from the Vault, it is potentially subject to manipulation via reentrancy. To call this function safely, attempt to trigger the reentrancy guard in the Vault by calling a non-reentrant function before calling `getActualSupply`.
+:::
+
+## Swap Fees
+
+### `getSwapFeePercentage`
+
+```
+getSwapFeePercentage() 
+returns (uint256)
+```
+
+Returns the current value of the swap fee percentage. Implemented in `ManagedPoolSettings`.
```

Comment:
This is generally true. To be more specific, it might be helpful to add that if the swap fee is undergoing a gradual update, it is the swap fee at the time of the last confirmed block